src/app.html

```diff
@@ -9,4 +9,14 @@
 	<body data-sveltekit-preload-data="hover" data-theme="sso-custom-theme">
 		<div style="display: contents" class="h-full overflow-hidden">%sveltekit.body%</div>
 	</body>
+	{% if env.GA_MEASUREMENT_ID %}
+		<script>
+			window.dataLayer = window.dataLayer || [];
+			function gtag(){dataLayer.push(arguments);}
+			gtag('js', new Date());
+
+			gtag('config', '{{ env.GA_MEASUREMENT_ID }}');
```

Comment:
env will throw an error, for vite env variables; read: https://vitejs.dev/guide/env-and-mode